script.js

```diff
@@ -151,28 +251,35 @@ function showMoneyQuestion(num) {
 function showTrueAnswers(event) {
   let target = event.target;
   let thisAnswers = document.querySelectorAll('.question-active .question__answers .answer__radio');
-  let falseAnswers = Array.from(thisAnswers).filter(answer => !correctAnswers.includes(answer.value));
-  let randomNum = function() {
-    return Math.floor(Math.random() * 3);
-  }
+  let falseAnswers = Array.from(thisAnswers).filter(answer => {
+    let falseAnswers = !correctAnswers.includes(answer.value) && !requestCorrectAnswers.includes(answer.value);
+    return falseAnswers && !answer.hasAttribute('disabled');
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/voronkova-ab/-game-Who-want-to-be-a-millionaire-/pullRequest?prid=7740148)
```suggestion
    return falseAnswers && !answer.hasAttribute("disabled");
```